docs/content/concepts/ops-jobs-graphs/op-events.mdx

```diff
@@ -26,13 +26,13 @@ It is also possible to raise Dagster-specific exceptions, which will indicate to
 
 ### Event Metadata
 
-Often, it may be useful to attach some arbitrary information to an event or exception that is not captured by its basic parameters. Through the <PyObject object="EventMetadataEntry"/> object, we provide a consistent interface for specifying this metadata on a variety of events. Depending on the type of the data, these entries will be rendered in Dagit in a more useful format than a simple unstructured string.
+Often, it may be useful to attach some arbitrary information to an event or exception that is not captured by its basic parameters. Through the <PyObject object="MetadataEntry"/> object, we provide a consistent interface for specifying this metadata on a variety of events. Depending on the type of the data, these entries will be rendered in Dagit in a more useful format than a simple unstructured string.
 
 The <PyObject object="AssetMaterialization" />, <PyObject object="AssetObservation" />, <PyObject object="ExpectationResult" />, and <PyObject object="Failure" /> objects each accept a `metadata` parameter, which maps string labels to structured values. <PyObject object="Output"/> also accepts this parameter, although this functionality is currently experimental may change in the future.
 
-We provide support for a wide variety of potentially useful metadata types, including simple datatypes (`EventMetadataEntry.float`, `EventMetadataEntry.int`, `EventMetadataEntry.text`), as well as more complex information such as markdown and json (`EventMetadataEntry.md`, `EventMetadataEntry.json`).
+We provide support for a wide variety of potentially useful metadata types, including simple datatypes (`MetadataEntry.float`, `MetadataEntry.int`, `MetadataEntry.text`), as well as more complex information such as markdown and json (`MetadataEntry.md`, `MetadataEntry.json`).
```

Comment:
Should these be `MetadataValue` instead of `MetadataEntry`?

Reply:
Probably eventually yes, but for now I'd say no. All of the docs are curretly written in terms of `MetadataEntry` (no mention of `MetadataValue`, previously `EventMetadataEntryData`. Because the `MetadataEntry` static API still works, the above is still valid.

When I post a `Metadata` concept page then I think it will be the time to update the various references to `MetadataEntry` or `MetadataValue`.

Reply:
If I understand correctly, we're telling users to use the `metadata` parameter, but then discussing types that can't be passed for that parameter.

Reply:
You're 100% right, I didn't read the preceding paragraph. I pushed a general rewrite of this section that hopefully clarifies things.

Reply:
Thanks for addressing. I realize that the issue I brought up preceded anything you changed.